test/integration/test-video-players-helper.js

```diff
@@ -213,6 +217,158 @@ export function runVideoPlayerIntegrationTests(
     afterEach(cleanUp);
   });
 
+  describe.configure().retryOnSaucelabs()
+  .run('Playing Status', function() {
```

Comment:
we should do these as unit tests in test-video-manager as the behaviour is not different between video-players. Also that would allow us to mock 'supportsAutoplay' so we can tests cases where `autoplay` is set but platform does not support it.